src/Board.coffee

```diff
@@ -61,6 +61,8 @@ class Board
     COLUMNS: BOARD_COLS
     WIDTH: BOARD_WIDTH
     HEIGHT: BOARD_HEIGHT
+  # HERE -> thinking about adding click disable flag...
+  @pauseCollectCoins = false
```

Comment:
I'm just curious, that while I realize that I think you are right in that I should move the '@pauseCollectCoins' to the constructor of the class, why do you say it has zero effect on the functionality of my code, if in the comment of the board clicked method you say that the if statement's condition requiring '@pauseCollectCoins' to be false _probably works like a charm_? Although, I will say that the disabling and enabling of input on the all coins collected condition I see needs work on my part